source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs

```diff
@@ -189,6 +191,76 @@ public IActionResult GetAcquisitionFileOwners([FromRoute] long id)
 
             return new JsonResult(_mapper.Map<IEnumerable<AcquisitionFileOwnerModel>>(owners));
         }
+
+        [HttpGet("{id:long}/project")]
```

Comment:
as discussed I believe these are returned from the acquisition file itself and are not necessary/redundant

Reply:
removed